tests/core/comparison_test.py

```diff
@@ -5,22 +5,24 @@
 import pytest
 import scipp as sc
 import numpy as np
-from ..common import assert_export
 
 
-def _check_comparison_ops_on(obj):
-    assert_export(obj.__eq__)
-    assert_export(obj.__ne__)
-    assert_export(obj.__lt__)
-    assert_export(obj.__gt__)
-    assert_export(obj.__ge__)
-    assert_export(obj.__le__)
+def test_comparison_operators():
+    assert not sc.scalar(5) < sc.scalar(3)
+    assert not sc.scalar(5) <= sc.scalar(3)
+    assert sc.scalar(5) > sc.scalar(3)
+    assert sc.scalar(5) >= sc.scalar(3)
+    assert not sc.scalar(5) == sc.scalar(3)
+    assert sc.scalar(5) != sc.scalar(3)
 
 
-def test_comparison_op_exports_for_variable():
-    var = sc.Variable(dims=['x'], values=np.array([1, 2, 3]))
-    _check_comparison_ops_on(var)
-    _check_comparison_ops_on(var['x', :])
+def test_comparison_functions():
+    assert sc.less(sc.scalar(1), sc.scalar(2))
+    assert sc.less_equal(sc.scalar(1), sc.scalar(2))
+    assert not sc.greater(sc.scalar(1), sc.scalar(2))
+    assert not sc.greater_equal(sc.scalar(1), sc.scalar(2))
+    assert not sc.equal(sc.scalar(1), sc.scalar(2))
+    assert sc.not_equal(sc.scalar(1), sc.scalar(2))
```

Comment:
Relying on `__bool__`? Is that good (it is basically testing two things in combination?)? I agree it is a bit of a chicken and egg problem.

Reply:
I could change it to use `sc.less(...).value`. But then you could argue that it also tests `.value`. We have to rely on something to do the test. Since we have dedicated tests for `__bool__`, I would say, it is ok to use it. But I can change the test if you want to.

Reply:
Leave it